### PR TITLE
Add service canary

### PR DESCRIPTION
**Pull request by xxx7xxxx**
Will make actions correct after https://github.com/megaease/easemesh-api/pull/16 merged

### CODE REVIEW

**Review by zhao-kun — APPROVED** [review]
LGTM

File: emctl/cmd/client/command/meshinstall/controlpanel/deploy.go [hunk 1]
```diff
@@ -160,6 +160,8 @@ func checkEasegressControlPlaneStatus(ctx *installbase.StageContext) error {
 		return errors.Wrap(err, "get mesh control plane entrypoint failed")
 	}
 
+	fmt.Printf("control plane endpoints: %+v\n", entrypoints)
```
Comment (localvar) [inline emctl/cmd/client/command/meshinstall/controlpanel/deploy.go:163]: is this line for debug only?

Comment (xxx7xxxx) [reply emctl/cmd/client/command/meshinstall/controlpanel/deploy.go:163]: More detailed install information

Comment (localvar) [reply emctl/cmd/client/command/meshinstall/controlpanel/deploy.go:163]: Got it

**Review by localvar — APPROVED** [review]

**Pull request merged**